src/app.py

```
@@ -6,6 +6,9 @@
 import sys
 import threading
 import uuid
+from collections import namedtuple
```

Ruff (0.14.0)

Redefinition of unused `namedtuple` from line 3

Remove definition: `namedtuple`

(F811)